docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+chown -R puppet:puppet /etc/puppetlabs/puppet/ssl
+chown -R puppet:puppet /opt/puppetlabs/server/data/puppetserver/
+
+if test -n "${PUPPETDB_SERVER_URLS}" ; then
+  sed -i "s@^server_urls.*@server_urls = ${PUPPETDB_SERVER_URLS}@" /etc/puppetlabs/puppet/puppetdb.conf
+fi
+
+# Configure puppet to use a certificate autosign script (if it exists)
+# AUTOSIGN=true|false|path_to_autosign.conf
+if [[ -n "${AUTOSIGN}" ]]; then
```

Comment:
Quotes especially not needed inside '[['

Also "test -n" is used above and "[[ -n " is used here. Probably better to choose one.

Reply:
ok, I'll take a look at the test / [[. This is a copy of a script from elsewhere, so there may be some wonkiness.